stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -163,6 +172,52 @@ enum AttributeCode {
   ///     operandTupleIndices: svarint[]
   ///   }
   kOutputOperandAlias = 14,
+
+  ///   CustomCallApiVersionAttr
+  ///     value: varint (encoded enum)
+  ///   }
+  kCustomCallApiVersionAttr = 15,
+
+  // Gap in numbers to separate StableHLO types from forked types //
```

Comment:
We don't. Was just in case we wanted a few spaces to keep StableHLO attrs next to eachother so StablehloBytecode and VhloBytecode numbers could resemble eachother for a short while.

Will remove the gap.